specs/AppLifecycle/EnvironmentVariables/AppLifecycle - Environment Variables (EV).md

```diff
@@ -10,12 +10,12 @@ Environment variables is one example, where there are several existing APIs
 that provide partially overlapping functionality, and with some gaps. This spec
 describes the Reunion support for environment variables.
 
-Important Note: this feature relies on other platform features which are only
-available from Windows summer 2021 onwards. Therefore this feature is also 
-only available from Windows summer 2021 onwards. An app can check the 
-IsSupported property to determine whether the current device supports the 
+Important Note: The EV tracking feature relies on other platform features which are only
+available from Windows 21H1+ (i.e. version >=10.0.19043.0). An app can check the 
+IsChangeTrackingSupported property to determine whether the current device supports the 
```

Comment:
```suggestion
IsChangeTrackingSupported property to determine whether the environment manager supports the 
```
The ability to do change tracking is a property of the environment, not of the device.